### ISSUE TITLE
As a advanced user I want to make mass upload of spots in order to shorter time for adding info on the map

### ISSUE THREAD

**Issue by rizomaa** [diglabby/mapa#9]
NTD

Who is "advanced user"?

I think the export function help us to focus on intaraction between platforms and accelerate thinking of API

**Comment by fr0zen** [comment]
We need mass upload functionality in case to import already existing data like OSM.

**Comment by fr0zen** [comment]
- [ ] Explore existing OSM export data format
- [ ] Consider, if it have useful information for us
- [ ] Find tools to easy extract data from OSM or write it.
- [ ] Import extracted data to openfairdb.

**Comment by rizomaa** [comment]
@fr0zen why do you think we need follow OSM format? Is it really in the scope of the task? The user story mostly about import data to Mapa (KVM) DB. Please explain.

**Comment by fr0zen** [comment]
@rizomaa i think that OSM is our main data provider about organisations. This is why we need to process format of data, that OSM gives us.

**Comment by wellemut** [comment]
Thats similar to #12 and #8

**Comment by rizomaa** [comment]
@wellemut yes, upload and download should be developed to give the chance make an opendata approach stronger here.

**Comment by wellemut** [comment]
Download is no problem (apart form Mail-adresses, which we should protect because of spam problems)

But upload needs a duplicate-Checker

**Comment by rizomaa** [comment]
@wellemut What is "a duplicate-Checker" ?

**Comment by Zverik** [comment]
Regarding OSM data format, you don't exactly need to parse it. Most likely you would use Overpass API, which gives an easy to use json.
